src/applications/my-education-benefits/components/EmailViewField.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export default function EmailViewField({ formData }) {
+  if (!formData) return 'not working';
```

Comment:
This should be returning an empty string or simply returning if a string isn't required.